.jenkins/pipelines/OpenEnclave/intel/tests/windows.Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '90',
+                artifactNumToKeepStr: '180',
+                daysToKeepStr: '90',
+                numToKeepStr: '180'
+            )
+        ),
+        [$class: 'JobRestrictionProperty'],
+        parameters(
+            [
+                string(name: "DCAP_URL", description: "Intel DCAP Package URL"),
+                string(name: "PSW_URL", description: "Intel PSW Package URL"),
+                string(name: "OECI_LIB_VERSION", defaultValue: 'release-pipeline', description: 'Version of OE Libraries to use'),
+                string(name: "OE_RELEASE_VERSION", defaultValue: "0.17.2", description: "Open Enclave Release Version"),
+                choice(name: "OE_PACKAGE", defaultValue: "open-enclave", choices: ["open-enclave", "open-enclave-hostverify"], description: "Open Enclave package type to install"),
+                choice(name: "RELEASE_SOURCE", defaultValue: "Github", choices: ["Azure", "GitHub"], description: "Source to download the OE Release from")
+            ]
+        )
+    ]
+)
+
+library "OpenEnclaveJenkinsLibrary@${params.OECI_LIB_VERSION}"
+
+parallel "Windows Server 2019": { tests.TestIntelRCs(globalvars.AGENTS_LABELS["acc-win2019-dcap"], params.OE_RELEASE_VERSION, params.OE_PACKAGE, params.RELEASE_SOURCE, false, params.DCAP_URL, params.PSW_URL, "-SkipVSInstall") }
```

Comment:
Does parallel with a single job work? I believe it fails when parallel is only supplied with 1 job, but it may have changed or not apply to scripted pipelines.

Reply:
Yeah, it works with a single job. Mind if I leave it like this to make it obvious that we want to add a second job in the future?

Reply:
Yeah, cool that this works!